### ISSUE TITLE
repo key download to fail

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#824]
https://github.com/openhab/openhabian/blob/master/functions/openhab.bash#L47

results in

```
2020-04-03_12:39:31_CEST [openHABian] Installing or upgrading to latest openHAB release (stable)... Warning: apt-key output should not be parsed (stdout is not a terminal)
gpg: no valid OpenPGP data found.
OK
2020-04-03_12:40:22_CEST [openHABian] Adding an openHAB dashboard tile for 'openhabiandocs'... Replacing... OK

```

See also #821 / #822 .

For me it works if executed manually on my openHABian box ...

@BClark09 any idea ?

**Comment by chaosjug** [comment]
I downloaded the current openhabian image and get this failure on first boot/install.
Is there a workaround for the issue?

**Comment by mstormi** [comment]
Github is used to coordinate development, not user issues. Please turn to the forum for support.

**Comment by mstormi** [comment]
Is anybody aware of any apt-key storage cache or similar, environment variable or other system conditions that could have the impact that it works on one box but not the other ? Short of 'Inet works' of course.

**Comment by mstormi** [comment]
Eventually related: https://community.openhab.org/t/openhab-2-5-x-patch-releases/90248/153

**Comment by mstormi** [comment]
Fixed by #835
